ip_list:
  - 192.168.0.1
  - 192.168.0.2

mac_list:
  - a1
  - a2

# Argos models always have 0 appended to their names
argos_models:
  - robot_0
  - robot_1

# this are the models to track
gazebo_models:
  - tb3_0
  - tb3_1

print_debug: true

net_use_uds: true
phy_use_uds: true

do_driver_transfer: false

netsim_uds_server_address: '/tmp/net_server_socket'

netsim_ip_server_address: 'localhost'
netsim_ip_server_port: 10000
netsim_ip_ranging_port: 10005

phy_uds_server_address: '/tmp/phy_server_socket'

phy_ip_server_address: 'localhost'
phy_ip_server_port: 10001

phy_driver_uds_server_address: '/tmp/phy_driver_socket'
net_driver_uds_server_address: '/tmp/net_driver_socket'

responsiveness_timeout: 2

packet_holding_duration: 50
driver_sync_time: 0.01

broadcast_address: '4294967295'  # 255.255.255.255 stored as string since yaml doesn't support long ints
